src/functions/template-tags/organizer.php

```diff
@@ -243,6 +243,11 @@ function tribe_get_organizer_link( $postId = null, $full_link = true, $echo = fa
 				return apply_filters( 'tribe_get_organizer_link', $link, $postId, $full_link, $url );
 			}
 		}
+
+		if ( ! $full_link ) {
```

Comment:
OK, but at the least, we should update the docs for that param. Right now it says:

> If true outputs a complete HTML <a> link, otherwise only the URL is output

With this change, it's more like:

> If true outputs a complete HTML <a> link, otherwise an empty string